src/main/java/se/lth/base/server/data/User.java

```diff
@@ -1,31 +1,96 @@
 package se.lth.base.server.data;
```

Comment:
Break out this class into its own branch

Reply:
I think this goes everything not related to the schema, so we first can merge that and then the schema

Reply:
Osäker på hur jag skulle göra det nu i efterhand, ändringarna där gjordes för att test skulle gå igenom. Jag började ändra i drive DTO också nu men jag lämnar resterande klasser till en annan branch.